### DISCUSSION TITLE
Support for JL1101 ethernet PHY on dingtian/dtwonder relay boards

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#18236] [Ideas]
On my dtwonder relay board ( https://nl.aliexpress.com/item/4001232791244.html ) is a JL1101 soldered but it is currently not supported by Tasmota.

For esphome this chip is already supported, so maybe this resource can help for implementing JL1101 in Tasmota.

https://github.com/esphome/esphome/pull/4009

![image](https://user-images.githubusercontent.com/2833940/226729107-1f4a7341-a914-49da-93b4-58b0c8f21ffc.png)

**Comment by barbudor** [comment]
@jeroenst This has alredy been discussed a lot in the original Dingtian thread around here: https://github.com/arendst/Tasmota/discussions/16797#discussioncomment-4504238

At some point the solution was to build a custom core/SDK but apprently @Jason2866 promised that would then be added in standard Tasmota core
So probably @Jason2866 and @DaniilMir and @hhorigian can help there

**Comment by Jason2866** [comment]
@jeroenst @barbudor The JL1101 PHY is supported in Tasmota and can be used in every tasmota32 esp32 build since release 12.4.0.

**Reply by jeroenst** [reply]
Which define do I have to enable? I searched for jl1101 in the sources but couldn't find it.

**Reply by Jason2866** [reply]
@jeroenst You don't have to enable anything. All there. Use this commands to setup https://tasmota.github.io/docs/Commands/#ethernet
The needed changes are in IDF and Arduino done.

The Tasmota driver is here https://github.com/arendst/Tasmota/blob/development/tasmota/tasmota_xdrv_driver/xdrv_82_esp32_ethernet.ino

**Reply by jeroenst** [reply]
Thank you!

**Comment by miky2k** [comment]


**Reply by Jason2866** [reply]
@miky2k Verified working with Dingtian DT-R008.
